worker/src/main/java/gov/cms/ab2d/worker/service/JobProcessingServiceImpl.java

```diff
@@ -30,28 +60,136 @@ public Job putJobInProgress(String jobId) {
         final Job job = jobRepository.findByJobUuid(jobId);
         Assert.notNull(job, String.format("Job %s not found", jobId));
 
-
         // validate status is SUBMITTED
         if (!SUBMITTED.equals(job.getStatus())) {
-            final String errMsg = String.format("Job %s is not in %s status. Skipping job", jobId, SUBMITTED);
+            final String errMsg = String.format("Job %s is not in %s status.", jobId, SUBMITTED);
             throw new IllegalArgumentException(errMsg);
         }
 
         job.setStatus(IN_PROGRESS);
 
-        log.info("Job [{}] is now IN_PROGRESS", job.getId());
         return jobRepository.save(job);
     }
 
     @Override
-    public void completeJob(Job job) {
+    public Job processJob(final String jobId) {
+
+        final Job job = jobRepository.findByJobUuid(jobId);
+        log.info("Found job : {} - {} - {} ", job.getId(), job.getJobUuid(), job.getStatus());
+
+        final Sponsor sponsor = job.getUser().getSponsor();
+
+        final List<Contract> attestedContracts = sponsor.getAttestedContracts();
+        log.info("number of attested contracts : {}", attestedContracts.size());
```

Comment:
Let's include job's UUID in this message as well. Will help traceability in Splunk.

Reply:
done